commands/scrape.ts

```diff
@@ -1,14 +1,16 @@
 import { BaseCommand } from '@adonisjs/core/ace'
 import type { CommandOptions } from '@adonisjs/core/types/ace'
-import { scrapeMenu } from '../scripts/scrapper.js'
+import { runScrapper } from '../scripts/scrapper.js'
```

Comment:
why `.js` ????

Reply:
<img width="1086" alt="image" src="https://github.com/user-attachments/assets/f0727673-4f0d-4ef0-b638-0d74aad40310">

i get this error when using `.ts`

Reply:
you can try to add this option in `tsconfig` I guess

Reply:
but Linek or Bartek prolly know more about this shit as I'm clueless

Reply:
why linking `.js` is bad option though?

Reply:
Idk, i just though it's some funny typo. If the types are preserved after the import then you can leave it i guess